pkg/notifiers/common.go

```diff
@@ -91,7 +91,7 @@ func wrapError(notifier string, resp *http.Response, errCode string) error {
 		if err != nil {
 			return errors.Wrapf(err, "Error reading %s response body", notifier)
 		}
-		log.Errorw("Received an error response for notifier",
+		logging.GetRateLimitedLogger().Errorw("Received an error response for notifier",
```

Comment:
If you want this log to be rate-limited, I think you need to use ErrorL, and pass something as limiter key (possibly the notifier parameter here).

Reply:
Sadly I think that's not enough, since the error messages are not 100% identical. In this case the rate limiter would still show all the logs, right?